test_utilities/bin/flutter_test_runner.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2019 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Runner for flutter tests. It expects a single parameter with the full
+# path to the flutter project where tests will be run.
+
+set -e
+
+echo "Running flutter tests from $1"
+pushd $1 > /dev/null
+
+flutter packages get
+# Remove the " || true" portion when https://github.com/flutter/flutter/issues/44370
+# is fixed.
+flutter analyze || true
```

Comment:
Flutter analyze doesn't throw an issue with the linked issue. Is there another issue luci runs into?

Reply:
info • Unused import: 'dart:ui' • lib/generated_plugin_registrant.dart:4:8 • unused_import and it finishes with a status code 1

Reply:
Is this doing a fresh install of the flutter app? We dealt with this on cirrus by running `flutter analyze` before any of the flutter scripts had run that would generate `lib/generated_plugin_registrant.dart`.

Reply:
This is just after running "flutter channel dev" and "flutter upgrade". Ideally from luci we will checkout the source code once and run tests in the different paths using a single bot.

Reply:
With the case of checking out the code once, we may just want to `rm lib/generated_plugin_registrant.dart` before running analyze. That was the alternative when we faced this issue on Cirrus.

Reply:
Isn't that file needed for anything in the flutter app?

Reply:
It's necessary for using plugins in the Flutter app, but for testing it's not necessary since all the plugins are mocked in testing. It's a side effect from building the app, and we build the app as another check to ensure master can be deployed at any time.

Reply:
"flutter analyze" fails with the same line even in a fresh checkout of flutter/cocoon is there a bug where we can keep the discussion? I still think this needs to get fixed rather than workarounded in infra.

Reply:
I think web plugins are still relatively WIP so this will change, but I filed an issue at https://github.com/flutter/flutter/issues/45711 for keeping track of this.

Reply:
Thanks, I found many other small details we need to address before starting to use these scripts. I'll open separate bugs.